client/gutenberg/extensions/slideshow/slideshow.js

```diff
@@ -49,26 +49,41 @@ class Slideshow extends Component {
 
 		return (
 			<div className={ className } data-effect={ effect }>
-				<div className="swiper-container" ref={ this.slideshowRef }>
+				<div
+					className="wp-block-jetpack-slideshow_container swiper-container"
+					ref={ this.slideshowRef }
+				>
 					<div className="swiper-wrapper">
 						{ images.map( ( { alt, caption, id, url } ) => (
-							<figure className="swiper-slide atavist-cover-background-color" key={ id }>
-								<div className="slide-background atavist-cover-background-color" />
-								<img
-									alt={ alt }
-									className="wp-block-slideshow-image-container"
-									data-id={ id }
-									src={ url }
-								/>
+							<figure className="wp-block-jetpack-slideshow_slide swiper-slide" key={ id }>
+								<div className="wp-block-jetpack-slideshow_image-container">
```

Comment:
Do we need the extra layer? Or can we style the `figure` instead?

Reply:
I wanted the `figcaption` to be the final element within the `figure`, and in order to center the `img` horizontally and vertically I needed to wrap it in an element. In the original version of the block based on `background-image` instead of `<img />` elements, this extra layer wasn't necessary.

Reply:
I tried messing with flexbox a bit:

```diff
diff --git a/client/gutenberg/extensions/slideshow/slideshow.js b/client/gutenberg/extensions/slideshow/slideshow.js
index 72e1e76078..4cfc868656 100644
--- a/client/gutenberg/extensions/slideshow/slideshow.js
+++ b/client/gutenberg/extensions/slideshow/slideshow.js
@@ -56,14 +56,12 @@ class Slideshow extends Component {
 					<div className="swiper-wrapper">
 						{ images.map( ( { alt, caption, id, url } ) => (
 							<figure className="wp-block-jetpack-slideshow_slide swiper-slide" key={ id }>
-								<div className="wp-block-jetpack-slideshow_image-container">
-									<img
-										alt={ alt }
-										className="wp-block-jetpack-slideshow_image"
-										data-id={ id }
-										src={ url }
-									/>
-								</div>
+								<img
+									alt={ alt }
+									className="wp-block-jetpack-slideshow_image"
+									data-id={ id }
+									src={ url }
+								/>
 								{ caption && (
 									<figcaption className="wp-block-jetpack-slideshow_caption">
 										{ caption }
diff --git a/client/gutenberg/extensions/slideshow/style.scss b/client/gutenberg/extensions/slideshow/style.scss
index aec579a8f4..e9b14cf867 100644
--- a/client/gutenberg/extensions/slideshow/style.scss
+++ b/client/gutenberg/extensions/slideshow/style.scss
@@ -7,6 +7,10 @@
 		height: 400px; // This is a default, which will be replaced programmatically
 	}
 	.wp-block-jetpack-slideshow_slide {
+		align-items: center;
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
 		margin: 0;
 	}
 	.wp-block-jetpack-slideshow_image-container {
```

Does that look about right?

(I'd be totally lost without the [froggies](https://flexboxfroggy.com/).)

Reply:
This approach raises a couple of issues related to captions. 

<img width="594" alt="screen shot 2019-02-06 at 9 52 51 am" src="https://user-images.githubusercontent.com/1477002/52349656-03439d80-29f5-11e9-9a75-0e392c2dba4a.png">

First of all, in the original design the images sit on a solid gray background and the caption is positioned below with no background. If we remove the wrapper then the background color can only be applied to the `<figure />` element, which places the caption on the background. Second, we need to account for the height of the caption or it will be cut off as shown in the screenshot. The caption styling is going to need some additional finessing anyway, and it's possible that we could figure out a solution to the height question without the container `<div />`, but the background color one is the trickier of the two.

Reply:
Okay. I guess let's leave it as-is for now so this PR isn't blocked by styles tweaking.

We can still revisit before releasing the block -- it's fine to change the `save()`d markup prior to release. (Afterwards, we should obviously strive to stabilize.)

Reply:
Filed #30629 so we can continue the conversation there.